runelite-client/src/main/java/net/runelite/client/plugins/statusbars/StatusBarsOverlay.java

```diff
@@ -0,0 +1,294 @@
+		{
+			return null;
+		}
+
+		final int maxHealth = client.getRealSkillLevel(Skill.HITPOINTS);
+		final int maxPrayer = client.getRealSkillLevel(Skill.PRAYER);
+		final int currentHealth = client.getBoostedSkillLevel(Skill.HITPOINTS);
+		final int currentPrayer = client.getBoostedSkillLevel(Skill.PRAYER);
+
+		Color healthBar = HEALTH_COLOR;
+		final int poisonState = client.getVar(VarPlayer.IS_POISONED);
+		if (poisonState > 0 && poisonState < 50)
+		{
+			healthBar = POISONED_COLOR;
+		}
+		else if (poisonState >= 1000000)
+		{
+			healthBar = VENOMED_COLOR;
+		}
+
+		final int counterHealth = client.getBoostedSkillLevel(Skill.HITPOINTS);
+		final int counterPrayer = client.getBoostedSkillLevel(Skill.PRAYER);
+		final String counterHealthText = Integer.toString(counterHealth);
+		final String counterPrayerText = Integer.toString(counterPrayer);
+		final Image healthImage = skillIconManager.getSkillImage(Skill.HITPOINTS, true);
+		final Image prayerImage = skillIconManager.getSkillImage(Skill.PRAYER, false);
+		final int quickPrayerState = client.getVar(Varbits.QUICK_PRAYER);
+		final Color prayerBar = quickPrayerState == 1 ? QUICK_PRAYER_COLOR : PRAYER_COLOR;
+
+		renderBar(g, offsetHealthX, offsetHealthY,
+				maxHealth, currentHealth, height, healthBar);
+
+		renderBar(g, offsetPrayerX, offsetPrayerY,
+				maxPrayer, currentPrayer, height, prayerBar);
+
+		if (config.enableRestorationBars())
+		{
+			int prayerHealValue = 0;
+			int foodHealValue = 0;
+			final MenuEntry[] menu = client.getMenuEntries();
+			final int menuSize = menu.length;
+			final MenuEntry entry = menu[menuSize - 1];
+			final Effect change = itemStatChanges.get(entry.getIdentifier());
+
+			if (change != null & entry.getParam1() == WidgetInfo.INVENTORY.getId())
+			{
+				final StatsChanges statsChanges = change.calculate(client);
+				for (final StatChange c : statsChanges.getStatChanges())
+				{
+					if (c.getStat().getName().equals("Hitpoints"))
+					{
+						foodHealValue = Integer.parseInt(c.getTheoretical());
+					}
+
+					if (c.getStat().getName().equals("Prayer"))
+					{
+						prayerHealValue = Integer.parseInt(c.getTheoretical());
+					}
+				}
+			}
+
+			renderHealingBar(g, offsetHealthX, offsetHealthY,
+					maxHealth, currentHealth, height,
+					foodHealValue, HEAL_COLOR);
+
+			renderHealingBar(g, offsetPrayerX, offsetPrayerY,
+					maxPrayer, currentPrayer, height,
+					prayerHealValue, PRAYER_HEAL_COLOR);
+		}
+
+		if (config.enableSkillIcon() || config.enableCounter())
+		{
+			renderIconsAndCounters(g, offsetPrayerX, offsetPrayerY, prayerImage.getScaledInstance(IMAGE_SIZE, IMAGE_SIZE, Image.SCALE_SMOOTH), counterPrayerText, PRAYER_LOCATION_X);
+			renderIconsAndCounters(g, offsetHealthX, offsetHealthY, healthImage, counterHealthText, HEALTH_LOCATION_X);
+		}
+
+		return null;
+	}
+
+	private void renderBar (Graphics2D graphics, int x, int y, int max, int current, int height, Color filled)
```

Comment:
This looks like it can be `static`?

```suggestion
	private static void renderBar (Graphics2D graphics, int x, int y, int max, int current, int height, Color filled)
```